src/components/library/IconStatus.tsx

```diff
@@ -0,0 +1,35 @@
+import { FunctionComponent } from "react";
+import { Stack, Theme } from "@mui/material";
+import { Typography } from "elements";
+import theme from "theme";
+
+export interface IconStatusProps {
+  readonly icon: JSX.Element;
+  readonly iconColor?: keyof Theme["colors"];
+  readonly fontColor?: keyof Theme["colors"];
+  readonly status: string | undefined;
+}
+
+/**
+ * Displays a vertically stacked icon and message.
+ */
+const IconStatus: FunctionComponent<IconStatusProps> = ({
+  icon,
+  iconColor = "music",
+  fontColor = "white",
+  status,
+}) => (
+  <Stack
+    display="flex"
+    spacing={ 1 }
+    direction="row"
+    color={ theme.colors[iconColor] }
+    sx={ { justifyContent: "center", alignItems: "center" } }
+  >
+    { icon }
+
+    <Typography color={ fontColor }>{ status }</Typography>
+  </Stack>
+);
```

Comment:
So the two edge cases I can think of are:

1. The status of the song is unable to query or error occurs feeding the status for some reason
Currently: By default `X Undistributed` would appear, which may be false based on the actual status

2. The icon errors and doesn't load, for either MUI or our custom icons 
Currently: The icon won't show up in the table but the string status should still load giving the user at least some awareness of status

To protect against the first item should I make it a strict matching case instead of a default case `X Undistributed`? Make the default case return null? I'm leaning toward this to not give the user false information and if there are truly errors with the server or queries than a null return would still tell them something is wrong by having missing information in the data field. Noting also that a recent change to the backend was made giving all songs a minting status on creation.

```
else if (mintingStatus === "Undistributed") {
    return (
      <IconStatus
        icon={ <Close fontSize="medium" sx={ { color: theme.colors.grey200 } } /> }
        fontColor="grey200"
        status="Undistributed"
      />
    );
  } else {
    console.log(mintingStatus);
    return null;
  }
```
Also just double checked and caught another issue on my end because of the above code. Not sure if there should be a better task handoff or tracking mechanism but @wlara updated the backend and changed the string return so the phrasing matched the design copy that the team finalized for Library 1.3, `"Undistributed"` from `"NotEarning"`, which I was still using in my conditional due to not being updated on the wiki and my own ignorance. However, because I was defaulting to `"Undistributed" ` nothing seemed wrong as a status was still appearing, obviously we would see complaints from users either way if status wasn't appearing due to a `null` return or a status appeared as `"Undistributed"` when the song was a different status. Not sure if that leads the preference a certain way for error/broken cases.

Reply:
Agree with `sx` prop consistency, I think I was having issue with color or something when first creating the component which led to the separation.